core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -42,7 +41,6 @@ import scala.reflect.ClassTag
 import scala.util.{Failure, Success, Try}
 import scala.util.control.{ControlThrowable, NonFatal}
 import scala.util.matching.Regex
-
```

Comment:
does these blank line is related to this change?

Reply:
This should be caused by opening the code specification.